nuxt.config.js

```diff
@@ -23,4 +24,99 @@ c.dir = {
   store:      path.posix.join(SHELL, 'store'),
 };
 
+c.build.extend = (config, { isClient, isDev }) => {
+  // Override the webpack target, so that we get the correct mix of
+  // electron (chrome) + nodejs modules (for ipcRenderer).
+  config.target = 'electron-renderer';
+  // Set a resolver alias for `./@pkg` so that we can load things from @ in CSS
+  config.resolve.alias['./@pkg'] = path.resolve(__dirname, 'pkg', 'rancher-desktop');
+
+  if ( isDev ) {
+    config.devtool = 'cheap-module-source-map';
+  } else {
+    config.devtool = 'source-map';
+  }
+
+  // if ( resourceBase ) {
```

Comment:
What's with the commented-out code?

Reply:
Good catch, this was left over from some testing.

Reply:
This appears to still exist.